.circleci/config.yml

```diff
@@ -117,29 +85,21 @@ workflows:
           name: << pipeline.parameters.job-name-deploy-staging >>
           stack_name: << pipeline.parameters.project-name >>-staging
           template: << pipeline.parameters.cluster-template >>
-          args: --capabilities CAPABILITY_NAMED_IAM --no-fail-on-empty-changeset --parameter-overrides Certificate=<< pipeline.parameters.certificate-staging >> Image=<< pipeline.parameters.repo >>/sharedid:<< pipeline.git.revision >> Environment=qa
+          args: --capabilities CAPABILITY_NAMED_IAM --no-fail-on-empty-changeset --parameter-overrides Certificate=<< pipeline.parameters.certificate-staging >> Image=<< pipeline.parameters.repo >>/sharedid:<< pipeline.git.revision >> Environment=qa CPU=256 Memory=512
```

Comment:
Is the CPU/Memory going to change per environment?

Reply:
Yes, production boxes are bigger (2048 CPU and 4G memory).

Reply:
Oh, cool.  That makes sense.